start.sh

```diff
@@ -5,4 +5,28 @@ if [ "$ENABLE_KERBEROS" = "yes" ]; then
   export DIVOLTE_JAVA_OPTS="-Djava.security.krb5.conf=/etc/krb5.conf -Dsun.security.krb5.debug=true"
 fi
 
+if [ -z "$DIVOLTE_HDFS_SINK_WORKING_DIR" ]; then
+	# No specific dir set use default and create it if not exists
+	if [ ! -d /tmp/working ]; then
+		mkdir -p /tmp/working
```

Comment:
`mkdir -p` does not really care if the directory already exists:
```
MacBook-Pro-van-Fokko:~ fokkodriesprong$ mkdir -p /tmp/
MacBook-Pro-van-Fokko:~ fokkodriesprong$ mkdir -p /tmp/
MacBook-Pro-van-Fokko:~ fokkodriesprong$ echo $?
0
```

Reply:
Thx. Done like suggested